dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.grpcMonitor-1.0.feature

```diff
@@ -0,0 +1,14 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.grpcMonitor-1.0
+Manifest-Version: 1.0
+IBM-API-Package: io.openliberty.grpc.monitor; type="ibm-api"
+IBM-Provision-Capability: \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.monitor-1.0))", \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=io.openliberty.grpc-1.0)(osgi.identity=io.openliberty.grpcClient-1.0)))"
+IBM-Install-Policy: when-satisfied
+-bundles=io.openliberty.grpc.1.0.internal.monitor, \
+  io.openliberty.grpc.1.0.internal, \
+  io.openliberty.grpc.1.0.internal.client
```

Comment:
I am guessing that this auto feature should only pull in `io.openliberty.grpc.1.0.internal.monitor`?